test/SizedArray.jl

```diff
@@ -125,6 +125,9 @@
         sa2 = SizedArray{Tuple{2, 2}, Int}([1, 2, 3, 4])
         @test (@inferred vec(sa2)) isa SizedVector{4, Int}
         @test vec(sa2).data === vec(sa2.data)
+
+        M = @MMatrix [1 2; 3 4]
+        @test copy(vec(M)) isa SizedVector{4, Int}
```

Comment:
A little unrelated to the original issue, but why does `vec(M::MMatrix)` return `SizedVector`? I thought it should be `MVector`. Is this documented behavior, or is there any performance reason for this?

Reply:
`vec` is supposed to return a view of the given matrix (see Julia's docs for `vec`) which `MVector` can't represent, that's why `SizedVector` is used.